OneCommunityGlobal/HighestGoodNetworkApp
# Communityportalaccess

# Description
The Access System is designed to provide secure, controlled entry to a secondary environment. This system mirrors the structure and functionality of Phase 2 and incorporates advanced security measures. Below are the main features and the modeled process:

## Related PRS (if any):
- Development Branch 
- No related PRs

## Main changes explained:
- Centralized Folder Structure:
All files related to Phase 3 are now organized under the CommunityPortal folder, streamlining development and deployment.

-CPDashBoard:
The DashBoard is moved into the CommunityPortal folder, serving as the central user interface for portal functionalities.

-CPLogin:
The login functionality specific to the CommunityPortal is encapsulated in the CPLogin module within the folder.

- Header:
The Header component, providing navigation and user actions, is now part of the CommunityPortal folder for consistency.

- Login Page:
The Login Page for the CommunityPortal is relocated to the folder, ensuring alignment with the portal's structure and styling.

## How to test:
1. check into current branch - "communityportalaccess"
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. Try http://localhost:3000/communityportal
5. log as admin user

## Screenshots or videos of changes:
<img width="1511" alt="Login Page for Community Portal" src="https://github.com/user-attachments/assets/a65b9c33-04b2-4472-8244-07141d660f5a" />


## Note:
Install All the required dependencies.
